src/basics.md

```diff
@@ -1770,6 +1771,48 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-read-environment-variable]: #ex-read-environment-variable
+<a name="ex-read-environment-variable"></a>
+## Read environment variable
+[![std-badge]][std] [![cat-filesystem-badge]][cat-filesystem]
+
+Set an environment variable and read it back. If the variable is not found, then fallback
```

Comment:
I'd suggest to link to std documentation for `std::env::set_var` and `std::env::var`